src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,210 @@
+    title="CLI Reference"
+    subtitle="Learn how to use the Daytona Command-Line Interface (CLI) to manage your development environments."
+    href="/reference/cli"
+    expanded
+  />
+</DocumentList>
+
+## Adding a Git Provider
+
+You can integrate Daytona with a Git Provider to manage version control operations and interact with your Git repositories.
+
+Daytona supports a variety of [Git Providers](/configuration/git-providers), including [GitHub](/configuration/git-providers#github), [GitLab](/configuration/git-providers#gitlab), [Bitbucket](/configuration/git-providers#bitbucket), and self-hosted solutions. This integration supports and enables interacting with your codebase, cloning repositories, and pushing changes directly from Daytona.
+
+<br />
+
+<CardGrid gridNumber={3} mobileGridNumber={1} colsMinWidth='150px' colsMinWidthMobile='100px'>
+    <TitleCard icon={githubLogo} title="GitHub" link="/configuration/git-providers#github" imageAlt="'GitHub logo"/>
+    <TitleCard icon={gitlabLogo} title="GitLab" link="/configuration/git-providers#gitlab" imageAlt="GitLab logo"/>
+    <TitleCard icon={bitbucketLogo} title="Bitbucket" link="/configuration/git-providers#bitbucket" imageAlt="BitBucket logo"/>
+</CardGrid>
+
+Use the `daytona git-providers --help` command to view available options for managing your Git Provider configurations and interacting with your repositories.
+
+<DocumentList>
+<DocumentListItem
+    title="Add a Git Provider"
+    subtitle="Learn how to add a Git Provider to Daytona and start managing your repositories effectively."
+    href="/configuration/git-providers#add-a-git-provider"
+    expanded
+  />
+</DocumentList>
+
+## Installing a Provider
+
+A [Provider](/configuration/providers) is a plugin that interfaces with the [Daytona Server](/usage/server), handling the deployment of the development environment and its lifecycle of management.
+
+Providers enable you to deploy your development environments with container management platforms and cloud hosting services.
+
+Follow the instructions provided in the [Providers installation](/configuration/providers#managing-providers) guide to install and manage a Provider. The guide includes detailed steps for installing Providers for various container management platforms and cloud hosting services.
+
+<DocumentList>
+<DocumentListItem
+    title="Install a Provider"
+    subtitle="Learn how to install a Provider and interface with the Daytona Server."
+    href="/configuration/providers#managing-providers"
+    expanded
+  />
+</DocumentList>
+
+## Setting a Target
+
+Once you have installed a Provider, you can set a [Target](/configuration/providers#managing-targets) to define how Daytona manages and deploys your development environment.
+
+Each Target is associated with a Provider and includes details for creating and managing your development environment in a specific location, such as a local machine, cloud server, or remote Docker instance.  
```

Comment:
also for @Tpuljak about definition of Target

A Target refers to the specific destination or environment where your development setups, facilitated by various providers, are deployed and managed. Providers define the method and technology used to create your environments, while targets specify the precise location or platform where these environments will reside. A target could be a local machine, a remote server, or a cloud instance, and it can vary based on the chosen provider.

Targets offer the flexibility to deploy and manage environments across different platforms and accounts, all within the unified interface